mtp_cashbook/settings/base.py

```diff
@@ -44,7 +44,7 @@
 )
 
 PROJECT_APPS = (
-    'core',
+    'moj_utils',
```

Comment:
Hm. The repo is ministryofjustice/django-utils (included in the `requirements.txt` as part of this change), but the app within it is `moj_utils`. @teneightfive I assume that's a “need to `pip install` again” type issue?